volplugin/volplugin.go

```diff
@@ -166,50 +166,56 @@ func (dc *DaemonConfig) getGlobal() {
 
 func (dc *DaemonConfig) watchGlobal() error {
 	for {
-		time.Sleep(1 * time.Minute)
+		time.Sleep(1 * time.Second)
 		dc.getGlobal()
 	}
 }
 
 func (dc *DaemonConfig) updateMounts() error {
-	cd := ceph.NewDriver()
-	mounts, err := cd.Mounted(dc.Global.Timeout)
-	if err != nil {
-		return err
-	}
+	for driverName := range backend.Drivers {
+		cd, err := backend.NewDriver(driverName, dc.Global.MountPath)
```

Comment:
@erikh I am just wondering if having a global mount-path work across drivers. Won't separate drivers step over each other, if they try to manage the same mount path?